styles.css

```diff
@@ -490,3 +405,69 @@ body {
 .description-item {
   color: #344563;
 }
+
+.contact-container {
+  background: url(Assets/contactmebg.png) no-repeat center, #6070ff;
+  background-position: 110px 20px;
+  padding-top: 100px;
+  border-top-left-radius: 100px;
+  display: flex;
+  width: 90%;
+  flex-direction: column;
+  height: 850px;
+  align-items: flex-start;
+  color: #fff;
+  text-align: center;
+}
+
+.c-title-container {
+  align-self: center;
+  font-size: 17px;
+  line-height: 20px;
+}
+
+.c-desc-container {
+  align-self: center;
+  width: 80%;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+}
+
+.contact-form {
+  align-self: center;
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  width: 65%;
+  text-align: center;
+  margin: 0 220px 50px 20px;
+}
+
+.customer-info {
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  color: #172b4d;
+}
+
+.customer-message {
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  font-size: 15px;
+  color: #b3bac5;
+}
+
+.submit {
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  cursor: pointer;
+  text-decoration: none;
+  padding: 10px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  margin: 0;
+}
```

Comment:
- [ ] According to the requirements, You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1), So kindly work on it 👍
- [ ] Kindly add button interactions such as hover, active as shown in the Figma template. 💯
- [ ] The form inputs should align center and the submit button should align left 💯
- [ ] Please, remove the extra spaces (margin or padding) on the right of the section 💯

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/192744378-8b6c864e-cd21-42dd-81a1-19439b10f3c3.png" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/192744523-96d5afc1-364a-4218-9f2b-d2b833652e95.png" width="500" /> |